src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -20,6 +20,7 @@ const SuggestedOwners = React.createClass({
       selector: '.tip',
       html: true,
       container: 'body',
+      template: '<div class="tooltip" role="tooltip"><div class="tooltip-arrow"></div><div class="tooltip-inner owners"></div></div>',
```

Comment:
it doesn't need to be jsx because this string gets injected into by bootstrap. it's the "template" option here: http://getbootstrap.com/javascript/#tooltips-options